mouse.go

```diff
@@ -5,6 +5,18 @@ import (
 	"github.com/luxengine/math"
 )
 
+type Cursor uint8
```

Comment:
Totally missed that one :P 

I basically moved some stuff around, so didn't think about documentation here.